ruby/vampires.rb

```diff
@@ -0,0 +1,126 @@
+vamp_con1 = 'Definitely a vampire!'
+
+vamp_con2 = '...Probably not a vampire.'
+
+vamp_con3 = 'Probably a vampire.'
+
+vamp_con4 = 'Almost certainly a vampire.'
+
+vamp_con5 = 'Results inconclusive.'
+
+
+puts 'How many employees will be processed?'
+
+employees_to_process = gets.chomp
+while employees_to_process.to_i > 0
+
+	puts 'What is your name?'
+	name = gets.chomp
+
+	if name.downcase == 'tu fang' || name.downcase == 'drake cula' || name.downcase == 'dracula' || name.downcase == 'nosferatu'
+		
+		vampire_name = true
+
+		else
+			vampire_name = false
+
+	end
+
+	puts 'How old are you?' 
+
+	age = gets.chomp
+
+	puts 'What year were you born?'
+
+	birthyear = gets.chomp
+
+	if age.to_i + birthyear.to_i == 2016 
```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.now.year`